tools/load_veldwerk_json.py

```diff
@@ -423,8 +425,52 @@ def fielddata_to_memcollections(filename, profile_plan_col=None, profile_id_fiel
 
             log.warning('records toegevoegd %i', i + 1)
 
+    # Extract fixed points from the geojson
+    for fp_pk, point_ids in enumerate(json_dict['point_notes']):
+
+        fp = json_dict['point_notes'][point_ids]
+        fixed_point = OrderedDict()
+
+        fixed_point['vp_pk'] = fp_pk
+        fixed_point['ids'] = fp.get('id', '')
+        fixed_point['project_id'] = project_id
+        fixed_point['proj_name'] = proj_name
+        fixed_point['type'] = fp.get('note_type', '')
+        fixed_point['opm'] = fp.get('remarks', '').replace('\n', '')
+
+        photo_list = fp.get('photos', None)
+        photo_string = ''
+
+        if photo_list:
+            for i in range(len(photo_list)):
+                f_name = os.path.basename(photo_list[i]).split('.')[0]
+                if i != (len(photo_list) - 1):
+                    photo_string = photo_string + (f_name + ";")
+                else:
+                    photo_string = photo_string + f_name
+
+        fixed_point['fotos'] = photo_string
+
+        fixed_point['datum'] = fp.get('datetime', '')
+
+        if fp.get('rd_coordinates') and not None in fp.get('rd_coordinates')[:2]:
+            rd_coordinates = fp.get('rd_coordinates')
+            fixed_point['x_coord'] = get_float(rd_coordinates[0])
+            fixed_point['y_coord'] = get_float(rd_coordinates[1])
+            if rd_coordinates[2] and rd_coordinates[2] != -99:
+                fixed_point['z'] = get_float(rd_coordinates[2])
+            else:
+                fixed_point['z'] = -9999
+            coordinates = (rd_coordinates[0], rd_coordinates[1])
+
```

Comment:
Wat gebeurt er met point_notes zonder coordinaten?

Reply:
Dat stukje was ik vergeten te programmeren inderdaad. Na overleg met Iris besloten ze wel te laten zien in de shapefile, met de coordinaten (0,0), waardoor het punt dus ergens in Frankrijk terecht komt. De attributen van de coordinaten in de tabel komen te staan als -9999 voor zowel x, y als z. Ik vind het niet een hele mooie oplossing eerlijk gezegd.
Iris zegt eigenlijk dat het gewoon überhaupt niet moet mogen voorkomen.

Reply:
is inderdaad niet mooi. QGIS kan bijvoorbeeld omgaan met None waarden, maar arcGIS niet.
z = null kan voorkomen als ze een punt op de kaart kiezen x en y kunnen null zijn als een punt wordt toegevoegd, terwijl er nog geen gps punt is ontvangen.